### PR TITLE
Akira 0.0.1 Alpha release

### PR DESCRIPTION
**Pull request by albfan**
https://github.com/akiraux/Akira

https://www.patreon.com/akiraux

Captura de pantalla de 2020-08-04 10-57-41

### CODE REVIEW

File: com.github.akiraux.akira.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      "/man",
+      "*.a",
+      "*.la"
+   ],
+   "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--talk-name=org.gtk.vfs.*",
+      /* dconf */
```
Comment (nedrichards) [inline com.github.akiraux.akira.json:23]: you shouldn't need this dconf snippet in apps based on 19.08 or 3.36 runtimes or later. Please remove.

**Comment by nedrichards** [comment]
bot, build com.github.akiraux.akira

**Comment by nedrichards** [comment]
Who from upstream will need to be added to the repo? Also, given the early stage that the app is at I assume you're after merging to the beta channel to begin with?

**Comment by albfan** [comment]
Ned probably beta is better for this. I just read something about that, will take another look

File: com.github.akiraux.akira.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+      "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```
Comment (LeandroStanger) [inline com.github.akiraux.akira.json:25]:
```suggestion
```

Comment (albfan) [reply com.github.akiraux.akira.json:25]: Sure, will do ASAP

**Comment by bilelmoussaoui** [comment]
bot, build com.github.akiraux.akira

**Pull request closed without merging**